io/eolearn/io/local_io.py

```diff
@@ -133,3 +134,68 @@ def execute(self, eopatch, *, filename):
             dst.write(output_array)
 
         return eopatch
+
+
+def _get_window(src, bbox):
+    x_res, y_res = src.transform[0], src.transform[4]
+    p_1 = Proj(bbox.get_crs().ogc_string())
+    p_2 = Proj(**src.crs)
+
+    # project mask boundaries from bbox CRS to source CRS
```

Comment:
I think ideally this would use the sentinel hub `BBox.transform` method but I wasn't able to get it to work with the CRS in this example

Reply:
I managed to use `BBox.transform` instead of directly using `pyproj` functions. However next steps, where you calculate a window, are not yet supported in sentinelhub-py.